### DISCUSSION TITLE
Dark Mode

### DISCUSSION THREAD

**Discussion by LFAdams** [snappysnappydog/Simple-IP-Config#175] [Ideas]
Is there any way to switch the app over to a dark mode? or plans to add this feature in the future? I absolutely love this program but it is one of the few things I use without a dark mode and I dont like getting blasted with white lite at 3am when i need to swap my IP.

**Comment by KurtisLiggett** [comment]
@LFAdams
That's a good Idea, I will see about adding this into the next version.

**Comment by KurtisLiggett** [comment]
Update: this feature has been implemented in the master branch, available in the next release v2.9.8.

Light Mode:
![image](https://user-images.githubusercontent.com/29669372/226236192-ef977d96-10e9-41e0-9f8d-8e59c340f628.png)

Dark Mode:
![image](https://user-images.githubusercontent.com/29669372/226236159-ad1585bc-b496-48e4-b715-44207507f83a.png)

**Reply by LFAdams** [reply]
This looks a lot easier on the eyes than the default! I think some of the lighter greys could still use a much darker color though, as they are still pretty white. Something dark enough that white text is required usually works the best for reducing light strain.
Thanks so much for listening to my idea!
![image](https://user-images.githubusercontent.com/54683715/226989061-93acacf6-306e-43e8-a217-d2e06880ad54.png)

**Comment by KurtisLiggett** [comment]
Also, I am open to any suggestions on color. I played with it for a while and came up with these as not too bright or dark.

**Comment by KurtisLiggett** [comment]
@LFAdams  How's this? Unfortunately, the menu bar color is set by Windows and I cannot change that directly...

![image](https://user-images.githubusercontent.com/29669372/226999226-d7a74817-87c8-4428-bc6a-597a2230ce06.png)

**Reply by LFAdams** [reply]
that one looks just about perfect!
